code/game/objects/structures/beds_chairs/alien_nest.dm

```diff
@@ -44,6 +44,8 @@
 					span_notice("You break free from the gelatinous resin!"),\
 					span_hear("You hear squelching..."))
 
+			M.remove_status_effect(/datum/status_effect/nest_sustenance)
```

Comment:
what if you get off via other methods?

Reply:
see what happens if you buckle to the thing, and then it gets deleted while you're on it. if it works as you want that's enough